jlexer/lexer.go

```diff
@@ -342,7 +342,12 @@ func (r *Lexer) fetchString() {
 	r.pos++
 	data := r.Data[r.pos:]
 
-	hasEscapes, length := findStringLen(data)
+	isValid, hasEscapes, length := findStringLen(data)
+	if !isValid {
+		r.pos += length
+		r.errParse("missing trailing string literal")
```

Comment:
"unterminated string literal", or "missing trailing quote char". A literal is the whole string with opening/ending quote chars.